core/src/main/java/com/linecorp/armeria/internal/server/DefaultServiceRequestContext.java

```diff
@@ -116,6 +116,7 @@
 
     @Nullable
     private String strVal;
+    private Boolean shouldLogUncaughtException;
```

Comment:
- Could be `boolean` instead of `Boolean`?
- How about renaming this property to `shouldLogException`? Exceptions are unexpected by their nature, so it's probably not worth adding 'unexpected'.

Reply:
Sounds like a great idea. 👍